ReactViewControl/IViewModule.cs

```diff
@@ -6,9 +6,9 @@ public interface IViewModule {
 
         string MainJsSource { get; }
 
-        string[] DependencyJsSources { get; }
+        string[] DependencyJsSources(bool useCache);
```

Comment:
You don't need to use this approach. You can modify ViewModuleContainer's Bind method to receive a parameter corresponding to the hot reload enabled. You should also set a Function in ViewModuleContainer that is invoked when we get DependencyJsSources (or CssSources). E.g.

```
private Function<string[]> func = DependencyJsSourcesCache.Value;

string[] IViewModule.DependencyJsSources => func.Invoke();

IViewModule.Bind(...) {
  if (hotReload) {
    func = () => GetDependenciesFromEntriesFile(... , hotReload: true);
  }
}
```

Reply:
This works for plugins as well